app/controllers/groups_controller.rb

```diff
@@ -546,6 +546,17 @@ def search
     render_serialized(groups, BasicGroupSerializer)
   end
 
+  def permissions
+    respond_to do |format|
+      format.html { render 'groups/show' }
```

Comment:
Curious why this is necessary? By default we bootstrap the app if the request is not an XHR and don't need to render anything specific.

Reply:
It must not be :)